tview/application.go

```diff
@@ -67,17 +68,17 @@ type Application struct {
 	// stop the application.
 	screenReplacement chan tcell.Screen
 
-	// Defines whether a bracketed paste is currently ongoing.
-	pasteActive bool
```

Comment:
Still needs a merge with master, this shouldn't be an outgoing deletion!

Reply:
Forgot to re-merge after reverting commits.